cmd/masa-node-cli/components.go

```diff
@@ -122,7 +122,7 @@ const logo = `
 const (
 	subtitle   = `masa oracle client`
 	navigation = `[yellow]use keys or mouse to navigate`
-	mouse      = `[green]v0.0.9-beta`
+	mouse      = `[green]v0.5.0`
```

Comment:
ditto here, we shouldn't bump the version here

Reply:
this was updated to reflect that version

Reply:
I get it, but It really should be a separate PR :)